docs/man-openmpi/man3/MPI_Status_get_error.3.rst

```diff
@@ -0,0 +1,71 @@
+.. _mpi_status_get_error:
+
+
+MPI_Status_get_error
+========================
+
+.. include_body
+
+:ref:`MPI_Status_get_error` |mdash| Retrieves the MPI_ERROR field from *status*.
+
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_Status_get_error(MPI_Status *status, int *error)
+
+
+Fortran Syntax
+^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE MPI
+   ! or the older form: INCLUDE 'mpif.h'
+   MPI_STATUS_GET_ERROR(STATUS, ERROR, IERROR)
+   	INTEGER	STATUS(MPI_STATUS_SIZE), IERROR
+   	INTEGER ERROR
+
+
+Fortran 2008 Syntax
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE mpi_f08
+   MPI_Status_get_error(status, error, ierror)
+   	TYPE(MPI_Status), INTENT(INOUT) :: status
+   	INTEGER, INTENT(OUT) :: error
+   	INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+
+
+INPUT/OUTPUT PARAMETER
+----------------------
+* ``status``: Status with which to retrieve error (status).
+
+INPUT PARAMETER
+---------------
+* ``error``: error to set in the MPI_ERROR field (integer).
+
+OUTPUT PARAMETER
+----------------
+* ``ierror``: Fortran only: Error status (integer).
+
+DESCRIPTION
+-----------
+
+Returns in error the MPI_ERROR field in the status object.
```

Comment:
```suggestion
Returns in error the MPI_ERROR field from the status object.
```

Reply:
also same as above. "from" has to be used for the _get, functions, "in" for the _set functions.